internal/e2e/main_test.go

```diff
@@ -1202,6 +1274,50 @@ func TestMinioPolicyBinding(t *testing.T) {
 
 		WaitForStableResourceVersion(td, pb)
 	})
+
+	t.Run("can still delete a minio policy binding when builtin user deleted", func(t *testing.T) {
```

Comment:
e2e tests that reproduces issues with deletion in `MinioPolicyBinding` resources and verifies that the issue is fixed